File: yangxj96-starter/yangxj96-starter-db/src/main/java/io/github/yangxj96/starter/db/properties/DBProperties.java
package io.github.yangxj96.starter.db.properties;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;

/**
 * db相关的props
 */
@Data
@ConfigurationProperties(prefix = "yangxj96.db")
public class DBProperties {

    /**
     * 是否启用
     */
    private boolean jdbcEnable = true;

    private boolean redisEnable = true;

    private Integer securityDb = 9;

}

File: yangxj96-starter/yangxj96-starter-db/src/main/kotlin/com/yangxj96/saas/starter/db/props/DBProperties.kt
package com.yangxj96.saas.starter.db.props

import org.springframework.boot.context.properties.ConfigurationProperties

/**
 * db相关的props
 */
@ConfigurationProperties(prefix = "yangxj96.db")
class DBProperties {

    /**
     * 是否启用
     */
    var jdbcEnable = true

    var redisEnable = true

    var securityDb = 0
}
